markdown/events/2021/bytesize-6-dsl2-using-modules-pipelines.md

```diff
@@ -34,6 +34,129 @@ This week, Friederike Hanssen ([@friederikehanssen](http://github.com/friederike
 
 The first talk will describe how to use DSL2 modules when writing an nf-core pipeline - both custom (`local`) modules, centralised from nf-core/modules and subworkflow files.
 
+<details markdown="1"><summary>Video transcription</summary>
+
+**Note: This text has been edited to make it more suitable for reading.**
+
+#### DSL2 - Using modules in a pipeline
+
+Thank you very much and welcome to the bytesize talk on how to use modules in a pipeline.
+
+I would just like to briefly recap what a module is. It is an atomic process that cannot be reduced any further and usually contains a single software tool like FastQC for example and it can be used within a pipeline and also shared between different pipelines. To make use of the sharing, there is an nf-core modules repository on GitHub where you can find many of these modules already.
+
+So to make use of this modules repository, there’s a new sub-command in nf-core/tools and for a brief recap, you can install it with ´pip install nf-core´ and ´conda install nf-core´ and if you then run ´nf-core modules´, you get a list of sub-commands that you can use to interact with this repository.
+
+I would just like to briefly recap what a module is. It is an atomic process that cannot be reduced any further and usually contains a single software tool like FastQC for example and it can be used within a pipeline and also shared between different pipelines. To make use of the sharing, there is an nf-core modules repository on GitHub where you can find many of these modules already.
+
+With one of the sub-commands, and with --help, you get some instructions on how to use them, so over the next couple of slides, I will briefly introduce you to some of the sub-commands that could be helpful for using modules in the pipeline. [0:47](https://youtu.be/tWvou0xj9wA?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=51)
+
+One of the first things you might want to do is to try and find a module that you could use, and you can use the ´nf-core modules list´ for that. [1:30](https://youtu.be/tWvou0xj9wA?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=90)
+
+That will just print out all the modules currently available and they are subdivided into tool and sub-tool because many tools like ´sam tools´ or ´bcf tools´ have these sub-commands that are then their own module basically.
```

Comment:
```suggestion
That will just print out all the modules currently available and they are subdivided into tool and sub-tool because many tools like ´samtools´ or ´bcftools´ have these sub-commands that are then their own module basically.
```